app/views/articles/index.html.erb

```diff
@@ -82,73 +83,11 @@
           <img src="<%= asset_path "lightning.svg" %>" alt="right-sidebar-nav">
         </button>
       </div>
+      <%# END Feed menu bar %>
 
-      <% if @featured_story.id %>
```

Comment:
As discussed yesterday @benhalpern, we`ll need to revert this change and only render this section if they are logged out.

Reply:
Done